be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/RollCallValidator.scala

```diff
@@ -57,10 +67,17 @@ case object RollCallValidator extends MessageDataContentValidator with EventVali
           EVENT_HASH_PREFIX, laoId.toString, data.opens.toString, data.opened_at.toString
         )
 
+        val sender: PublicKey = message.sender
+        val channel: Channel = rpcMessage.getParamsChannel
+
         if (!validateTimestampStaleness(data.opened_at)) {
           Right(validationError(s"stale 'opened_at' timestamp (${data.opened_at})"))
         } else if (expectedRollCallId != data.update_id) {
           Right(validationError("unexpected id 'update_id'"))
+        } else if (!validateOwner(sender, channel)){
+          Right(validationError(s"invalid sender $sender"))
+        } else if (!validateChannelType(ObjectType.LAO, channel)) {
```

Comment:
Maybe add a comment here as a reminder of why we're using `ObjectType.LAO` rather than `ObjectType.RollCall`. I personnaly gets confused ever time

Reply:
I'll add it here too, however, it is already stated in some parts of the code